src/sentry/static/sentry/app/components/events/eventEntries.jsx

```diff
@@ -104,7 +105,13 @@ class EventEntries extends React.Component {
   renderEntries() {
     const {event, project, isShare} = this.props;
 
-    return event.entries.map((entry, entryIdx) => {
+    const entries = event && event.entries;
+
+    if (!isArray(entries)) {
```

Comment:
Do we need lodash `isArray` vs the more commonly used `Array.isArray` for some reason?

Reply:
Old habits die hard 😆 

I've now changed it to  `Array.isArray`.